monitor_packages/postgresql/README.md

```diff
@@ -0,0 +1,86 @@
+# sumologic-postgresql-monitors
```

Comment:
This readme will move to Sumo logic help, we just need to point to Sumo Help, check redis or nginx.

Reply:
done